### ISSUE TITLE
Add additional fields to status endpoint

### ISSUE THREAD

**Issue by bio-boris** [kbase/execution_engine2#43]
```
{"gitcommithash":"a2ed4cb88120aeb10a295919cb0fba85e13d462d","servertime":1568823552860,"service":"Notification Feeds Service","version":"1.0.2"}
```

@briehl

**Comment by briehl** [comment]
Yup. Just need to make those pieces of info available. Shouldn't be hard.

There's others in the .spec file, but they might be overkill? Or maybe should be separate endpoints, like getting the queued job counts for all users. Though it might make sense to have the total queue state in this call (or maybe a separate one).
